.eslintrc

```diff
@@ -1,14 +1,7 @@
 {
-  "extends": "airbnb",
```

Comment:
I'd just ignore the root project stuff. Though you might be interested in a discussion around whether to use airbnb linter rules or not. You might even note that I've gone sour on semi colons.

Reply:
Also agree with exporting linter and prettier rules (BE are doing this). Re configs, if they're consistent, I don't mind (although I prefer semi-colons). I think the best way to go is just use defaults across the board, just because it stops any disagreements.